service/frontend/nexus_handler.go

```diff
@@ -38,6 +38,9 @@ import (
 	nexuspb "go.temporal.io/api/nexus/v1"
 	"go.temporal.io/api/serviceerror"
 	"go.temporal.io/api/taskqueue/v1"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
```

Comment:
nit: I see this was already the case but we agreed not to separate temporal from non-temporal imports as a team. If you have some config in your IDE that adds this separation, make sure to remove it.

Reply:
Yeah I think I removed my IDE settings but it does not automatically combine groups that are separated. Updated this file now.